README.md

```diff
@@ -11,111 +16,148 @@
-  <a href="https://github.com/cabaletta/baritone/pulls/"><img src="https://img.shields.io/github/issues-pr/cabaletta/baritone.svg" alt="Pull Requests"/></a>
-  <a href="https://github.com/cabaletta/baritone/graphs/contributors/"><img src="https://img.shields.io/github/contributors/cabaletta/baritone.svg" alt="GitHub contributors"/></a>
-  <a href="https://github.com/cabaletta/baritone/commit/"><img src="https://img.shields.io/github/commits-since/cabaletta/baritone/v1.0.0.svg" alt="GitHub commits"/></a>
-  <img src="https://img.shields.io/github/languages/code-size/cabaletta/baritone.svg" alt="Code size"/>
-  <img src="https://img.shields.io/github/repo-size/cabaletta/baritone.svg" alt="GitHub repo size"/>
-  <img src="https://tokei.rs/b1/github/cabaletta/baritone?category=code" alt="Lines of Code"/>
-</p>
+#### Baritone is a Minecraft [pathfinding](https://en.wikipedia.org/wiki/Pathfinding) bot.
 
-<p align="center">
-  <a href="https://impactclient.net/"><img src="https://img.shields.io/badge/Impact%20integration-v1.2.14%20/%20v1.3.8%20/%20v1.4.6%20/%20v1.5.3%20/%20v1.6.3-brightgreen.svg" alt="Impact integration"/></a>
-  <a href="https://github.com/kami-blue/client"><img src="https://img.shields.io/badge/KAMI%20Blue%20integration-v1.2.14--master-green" alt="KAMI Blue integration"/></a>
-  <a href="https://github.com/fr1kin/ForgeHax/"><img src="https://img.shields.io/badge/ForgeHax%20%22integration%22-scuffed-yellow.svg" alt="ForgeHax integration"/></a>
-  <a href="https://aristois.net/"><img src="https://img.shields.io/badge/Aristois%20add--on%20integration-v1.6.3-green.svg" alt="Aristois add-on integration"/></a>
-  <a href="https://rootnet.dev/"><img src="https://img.shields.io/badge/rootNET%20integration-v1.2.14-green.svg" alt="rootNET integration"/></a>
-  <a href="https://futureclient.net/"><img src="https://img.shields.io/badge/Future%20integration-v1.2.12%20%2F%20v1.3.6%20%2F%20v1.4.4-red" alt="Future integration"/></a>
-  <a href="https://rusherhack.org/"><img src="https://img.shields.io/badge/RusherHack%20integration-v1.2.14-green" alt="RusherHack integration"/></a>
-</p>
+#### Baritone currently supports Minecraft versions 1.12.2 - 1.16.5
 
-<p align="center">
-  <a href="http://forthebadge.com/"><img src="https://forthebadge.com/images/badges/built-with-swag.svg" alt="forthebadge"/></a>
-  <a href="http://forthebadge.com/"><img src="https://forthebadge.com/images/badges/mom-made-pizza-rolls.svg" alt="forthebadge"/></a>
-</p>
+Baritone has been used in popular utility clients such as [Impact Client](https://impactclient.net/),
+[Meteor Client](https://meteorclient.com/), and [Aristois](https://aristois.net/).
+
+#### You can install Baritone from the following locations:
 
-A Minecraft pathfinder bot.
+- [This repository](https://github.com/cabaletta/baritone/releases/)
+- [Impact Client](https://impactclient.net/)
+- [Meteor Client](https://meteorclient.com/)
+- [Aristois](https://aristois.net/)
+  
```

Comment:
not sure about policy on mentioning utility mods...

Reply:
Except impact these shouldn't be mentioned by name.

Impact being mentioned is @leijurv's choice, I don't know if they want to keep the strong link between impact and baritone that there currently is.

Reply:
well they all have icons in the current readme... 
![image](https://user-images.githubusercontent.com/6234704/125345716-ba9f1580-e30d-11eb-91e1-464e80f1bb30.png)

Reply:
Icons are fine, I especially when it mentions many utility client, I just feel like that this is almost endorsing these clients, something which I feel the icons do much less than having actual text.

Reply:
yeah. that's the general feeling I got as well...